python/clean_export_aggregate.py

```diff
@@ -48,7 +48,6 @@ def get_index_columns(name: str):
     aggregate.set_index(idx, inplace=True, verify_integrity=True)
 
     df_concat = pd.concat([geo_base, aggregate], axis=1)
-    final = df_concat.loc[:, ~df_concat.columns.duplicated()].copy()
-    final.fillna(value=0, inplace=True,)
+    final = df_concat.fillna(value=0)
```

Comment:
Doesn't this still leave duplicate columns for the NTA names? I imagined we would add code to drop the NTA name column in the aggregate dataframe

Reply:
I remembered after your comment I already took them out in the aggregate table so it is none issue now. The output on digital ocean looks good! https://cloud.digitalocean.com/spaces/edm-publishing?i=266877&path=db-developments%2F572-comprehensive-records%2Flatest%2Foutput%2F